src/index.ts

```diff
@@ -79,31 +84,91 @@ async function setDebugSession(
   app.commands.notifyCommandChanged();
 }
 
+function updateToolbar(
+  widget: NotebookPanel | ConsolePanel | DocumentWidget,
+  debug: IDebugger
+): void {
+  if (!debug.isDebuggingEnabled) {
+    return;
+  }
+  const isConsolePanel = widget instanceof ConsolePanel;
+
+  const changeAtribute = () => {
+    if (debug.session.debuggedClients.has(debug.session.client.name)) {
+      widget.node.setAttribute('data-jp-debugger', 'true');
+    } else {
+      widget.node.removeAttribute('data-jp-debugger');
+    }
+  };
+
+  const getToolbar = (): Toolbar => {
+    if (isConsolePanel) {
+      return (
+        ((widget as ConsolePanel).widgets.find(
+          widget => widget instanceof Toolbar
+        ) as Toolbar) || new Toolbar()
+      );
+    } else {
+      return (widget as NotebookPanel | DocumentWidget)?.toolbar;
+    }
+  };
```

Comment:
Could something like this be more readable? Also it makes it more apparent that `ConsolePanel` is (for now) a special case:

```typescript
  const getToolbar = (): Toolbar => {
    if (!(widget instanceof ConsolePanel)) {
      return widget.toolbar;
    }
    const toolbar = widget.widgets.find(w => w instanceof Toolbar) as Toolbar;
    return toolbar ?? new Toolbar();
  };
```